### PR TITLE
Create MainWindow

### PR DESCRIPTION
**Pull request by cymes1**
The main window is a starting point of the tool.

### CODE REVIEW

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/MainWindow.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+using UnityEditor;
+using UnityEngine;
+using Zenject;
+
+namespace PushForce.FastProjectInitializer.UI
+{
+	public class MainWindow : ZenjectEditorWindow
+	{
+		public const string TITLE = "Fast Project Initializer";
```
Comment (Snaki94) [inline fast-project-initializer/Assets/_Scripts/Editor/UI/Views/MainWindow.cs]: Create and move to keys class

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+namespace PushForce.FastProjectInitializer.UI
+{
+	public class DirectoryInitializerView : IView
+	{
+		public void drawGUI()
+		{
+			GUI.Label(new Rect(25, 75, 200, 50), "Test Label");
+			if (GUI.Button(new Rect(25, 25, 200, 50), "Restart"))
```
Comment (Snaki94) [inline fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs:10]: keys

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+			{
+				Debug.Log("click");
+			}
+		}
```
Comment (Snaki94) [inline fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs:14]: The method name should start with a capital letter

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/Interfaces/IView.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+namespace PushForce.FastProjectInitializer.UI
+{
+	public interface IView
+	{
+		void drawGUI();
```
Comment (Snaki94) [inline fast-project-initializer/Assets/_Scripts/Editor/UI/Views/Interfaces/IView.cs:5]: same here

**Review by Snaki94 — CHANGES_REQUESTED** [review]

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/Interfaces/IView.cs (hunk 4 above)
Comment (cymes1) [reply fast-project-initializer/Assets/_Scripts/Editor/UI/Views/Interfaces/IView.cs:5]: It's fixed on another branch.

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs (hunk 3 above)
Comment (cymes1) [reply fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs:14]: It's fixed on another branch.

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs (hunk 2 above)
Comment (cymes1) [reply fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs:10]: Those keys will disappear in the future so i will leave them as they are for now.

File: fast-project-initializer/Assets/_Scripts/Editor/UI/Views/MainWindow.cs (hunk 1 above)
Comment (cymes1) [reply fast-project-initializer/Assets/_Scripts/Editor/UI/Views/MainWindow.cs]: done

**Review by Snaki94 — APPROVED** [review]
LGTM

**Pull request merged**